storage-local/src/main/java/io/kestra/storage/local/LocalStorage.java

```diff
@@ -94,6 +110,26 @@ public URI put(URI uri, InputStream data) throws IOException {
         return URI.create("kestra://" + uri.getPath());
     }
 
+    @Override
+    public URI put(URI uri) throws IOException {
+        File file = Paths.get(config.getBasePath().toAbsolutePath().toString(), uri.getPath()).toFile();
+        if (!file.exists()) {
+            if (!file.mkdirs()) {
+                throw new RuntimeException("Cannot create directory: " + file.getAbsolutePath());
+            }
+        }
+        return URI.create("kestra://" + uri.getPath());
+    }
+
+    @Override
+    public URI rename(URI from, URI to) throws IOException {
+        Files.move(
+            Paths.get(config.getBasePath().toAbsolutePath().toString(), from.getPath()),
+            Paths.get(config.getBasePath().toAbsolutePath().toString(), to.getPath()),
```

Comment:
As Loïc said this morning I think we should add a prefix for namespace files like files/

Reply:
hmm question is should this be done inside the storage ? To me prefix will be a matter of the storage "user" right ? so we should add a default in the storage interface to retrive a prefix for file storage maybe ?